Allow for custom css sheet overrides for reports (#1573)
ewels/MultiQC: docs/config.md

produce a huge report file with all of the embedded plot data and crash your browser when opening it.
If you are running MultiQC for the `multiqc_data` folder and never intend to look at the report, it
speed things up though.

## Custom CSS files

> MultiQC generates an html report. You can include custom CSS in your final report.

Reviewer: ```suggestion
MultiQC generates HTML reports. You can include custom CSS in your final report if you wish.
```

> Simply add css files to the `custom_css_files` config option:

Reviewer: ```suggestion
Simply add CSS files to the `custom_css_files` config option:
```

```yaml
custom_css_files:
  - myfile.css
```

> Or pass `--custom-css-files` (multiple params accepted) and multiqc will include

Reviewer: ```suggestion
Or pass `--custom-css-files` (can be specified multiple times) and MultiQC will include
```

> them in the final report html.

Reviewer: ```suggestion
them in the final report HTML.
```
